awx/main/utils/common.py

```diff
@@ -118,7 +118,7 @@ class RequireDebugTrueOrTest(logging.Filter):
 
     def filter(self, record):
         from django.conf import settings
-        return settings.DEBUG or 'test' in sys.argv
+        return settings.DEBUG or settings.IS_TESTING()
```

Comment:
This is the main fix. When running tests, `settings.DEBUG` happens to be `False`. Since our tests are invoked with `py.test`, the string "test" is not present in `sys.argv`.

Thus, our logging configuration for testing essentially has logging turned off, for this unintended reason.